Jenkinsfile

```diff
@@ -43,16 +43,16 @@ spec:
         }
     }
     stages {
-        stage('Diagnostics') {
-            steps {
-                container('docker') {
-                    sh "printenv; id; cat /etc/passwd"
-                }
-                container('node') {
-                    sh "printenv; id; cat /etc/passwd"
-                }
-            }
-        }
+        // stage('Diagnostics') {
```

Comment:
these add noise in the jenkins console, that though could be useful while troubleshooting, are not necessary for most builds